src/task.js

```diff
@@ -0,0 +1,41 @@
+export const allTasks = document.querySelector('.task-list');
+let todos = [];
+
+export function getTodos() {
+  if (localStorage.getItem('todos') === null) {
+    todos = [];
+  } else {
+    todos = JSON.parse(localStorage.getItem('todos'));
+  }
+  let newDisplay = '';
+  todos.forEach((item) => {
+    const check = item.completed ? 'checked' : '';
+    const completed = item.completed ? 'completed' : '';
+    newDisplay += `
+    <li ${completed}>
+    <input type="checkbox" data-id="${item.index}" name="" class="check-list" ${check}>
+    <label class="list ${completed}" contentEditable = "true" for="">${item.description}<i class="fas fa-edit column"></i><i class="far fa-trash-alt trash"></i></label>
+    </li>
+    `;
+  });
+  allTasks.innerHTML = newDisplay;
+}
+getTodos();
+
+export function displayBox() {
```

Comment:
- [ ] <b>Optional:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```